cpp/src/arrow/python/pandas_convert.cc

```diff
@@ -176,6 +178,35 @@ Status AppendObjectStrings(StringBuilder& string_builder, PyObject** objects,
   return Status::OK();
 }
 
+Status AppendObjectFixedWidthBytes(FixedWidthBinaryBuilder& builder, PyObject** objects,
```

Comment:
Yeah, this was following the conventions of `AppendObjectStrings`. I'll change both.